linux_os/guide/system/network/network_sniffer_disabled/rule.yml

```diff
@@ -22,8 +22,6 @@ rationale: |-
 
 severity: medium
 
-platform: machine  # The oscap interface probe doesn't support offline mode
```

Comment:
Packages can be installed inside containers too. Reverting.

Reply:
This doesn't have anything to do with package installed. The `interface_test` doesn't make sense in offline mode.